package.json

```diff
@@ -49,19 +49,19 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
-    "@fontsource/inter": "^5.0.18",
+    "@fontsource/inter": "^5.0.20",
     "@glennsl/bs-json": "^5.0.4",
     "@googlemaps/react-wrapper": "^1.1.35",
     "@googlemaps/typescript-guards": "^2.0.3",
     "@headlessui/react": "^1.7.15",
     "@pnotify/core": "^5.2.0",
     "@pnotify/mobile": "^5.2.0",
-    "@react-spring/web": "^9.7.3",
+    "@react-spring/web": "^9.7.4",
```

Comment:
I don't think we are using this anymore https://github.com/coronasafe/care_fe/pull/8306